cypress/integration/models/administrator/repositories/maven.ts

```diff
@@ -27,10 +27,18 @@ export class MavenConfiguration {
     clearRepository() {
         MavenConfiguration.open();
         cy.wait(2000);
-        clickByText(clearRepository, "Clear repository");
 
-        cy.get(".pf-c-modal-box__title-text").contains("Clear repository");
+        cy.get("#clear-repository")
```

Comment:
You can use the clearRepository constant defined in 

`cypress/integration/views/repository.view.ts`